cell_test.go

```diff
@@ -176,6 +176,18 @@ func TestSetCellFloat(t *testing.T) {
 	assert.EqualError(t, f.SetCellFloat("Sheet:1", "A1", 123.42, -1, 64), ErrSheetNameInvalid.Error())
 }
 
+func TestSetLongUtf8CellValues(t *testing.T) {
```

Comment:
- I suggest using the `TestSetCellValuesMultiByte` function instead of `TestSetLongUtf8CellValues`, so we can run test with all set cell value related cases which prefix by `TestSetCellValues*`
- Using the unicode represent multi byte character
- Using the `value` instead of `long_value`

For example:

```go
value := strings.Repeat("\u4E00", TotalCellChars+1)
```

Reply:
Done